004-governance.Rmd

```diff
@@ -0,0 +1,153 @@
+
+### Users
+
+People who use the package are the most important members of the community; without these users, this project would have no purpose.
+Users are encouraged to participate in the life of the project and the community as much as possible.
+Common user activities include (but are not limited to):
+
+-   Evangelising about the project.
+-   Asking and answering questions on community forums.
+-   Providing moral support (a "thank you" goes a long way).
+
+### Contributors
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved.
+Such users may then go on to become **contributors** by interacting with the project on GitHub.
+Contributors:
+
+-   Report bugs and suggest improvements by creating new issues.
+
+-   Improving existing issues by answering questions, creating reprexes, or providing feedback on proposed changes.
+
+-   Contributing code or documentation via pull requests.
+
+Anyone can become a contributor: there is no expectation of commitment to the project, no required set of skills, and no selection process.
+The only requirement is to follow the code of conduct.
+
+Packages don't maintain an explicit list of contributors but acknowledge them in blog posts using data from GitHub aggregated by `usethis::use_tidy_thanks()`.
+
+### Authors
+
+Contributors who have made significant and sustained contributions can be invited to become authors.
+Authors are collectively responsible for day-to-day development of the package, including responding to issues and reviewing pull requests.
+An author possesses two special powers:
+
+-   They have **write** access on GitHub so they can triage issues, request review on PRs, and merge them.
+
+-   They are listed in `Authors@R` so they receive credit when others cite the package.
+
+Authors are expected to follow our standard processes, such as:
+
+-   **Code contribution**: code is usually contributed via PR, even for authors who could push directly.
+    Particularly high-stakes project may want to protect the main branch and require "request reviews before merging".
+
+-   **Communication**: authors are involved in most of the interactions with contributors and thus need to set a welcoming and inclusive tone for the project.
+
+-   **PR review**: all pull requests should be reviewed by at least one other author.
+    In general, there is no expectation that PRs contain clean commit histories, but it's appreciated where possible.
+
+-   **Backward compatibility**: any backward incompatible changes (i.e. changes that cause reverse dependencies to fail `R CMD check` or are likely to cause problems in user code) must be approved by the maintainer.
+    Significant backward incompatible changes need to be accompanied with a plan for how they will be communicated to the community.
```

Comment:
It feels like the lifecycle system/package should come up here. Is it intentional to *not* be quite that specific?

Reply:
Right, this is just a high-level list of important stuff. It might go away altogether in favour of the longer process doc.